tests/test_dev.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 import click
 import click.testing
+import git
```

Comment:
Unused?

Reply:
man i am so use to pycharm... maybe we can make a pre-commit hook to clean up unused imports?

307863c